.github/workflows/python.yml

```diff
@@ -42,8 +42,9 @@ jobs:
       run: pip install .[dev]
     - name: Test with pytest
       run: pytest --cov=mesa tests/ --cov-report=xml
-    - name: Codecov
-      uses: codecov/codecov-action@v1.0.15
+    - if: matrix.os == 'ubuntu'
```

Comment:
Should we limit it to 1 Python version on Ubuntu, instead of multiple Python version on Ubuntu? Probably the oldest supported version is prioritized.

Reply:
It seems to be recommended to upload them al, since they can differ between versions. And this PR already reduces the number of reports uploaded.

https://community.codecov.com/t/upload-of-coverage-of-several-python-versions/2349